mantle/kola/tests/ignition/luks.go

```diff
@@ -50,6 +51,17 @@ func init() {
 		ExcludeArchitectures: []string{"s390x"}, // no TPM backend support for s390x
 		Tags:                 []string{"luks", "tpm", "tang", "sss", kola.NeedsInternetTag, "reprovision"},
 	})
+	register.RegisterTest(&register.Test{
+		Run:           runCexTest,
+		ClusterSize:   0,
+		Name:          `luks.cex`,
+		Description:   "Verify that CEX-based rootfs encryption works.",
+		Flags:         []register.Flag{},
+		Distros:       []string{"rhcos"},
```

Comment:
This should in theory work on FCOS, it's just that we don't have a CEX device on the FCOS builder currently, right? So I think it'd be more accurate to not limit to RHCOS here, and just let the test be skipped on FCOS.

Reply:
I mean to just remove `Distros` entirely. The test should be skipped when running on the FCOS s390x builder (where we don't have a CEX device).